src/v/cluster/tx_registry_frontend.h

```diff
@@ -25,6 +26,14 @@ namespace cluster {
 class tx_registry_frontend final
   : public ss::peering_sharded_service<tx_registry_frontend> {
 public:
+    struct info {
+        repartitioning_id id;
+        absl::flat_hash_map<model::partition_id, tx_registry_stm::hosted_txs>
+          mapping;
```

Comment:
Do we want to copy this potentially large data structure for observability  ?

Reply:
by default it's safe to pass by value and in this case the size of the struct depends on the admin actions and not on the user actions so it won't be large.